hack/generator/pkg/astmodel/package_import_set.go

```diff
@@ -6,12 +6,11 @@
 package astmodel
 
 import (
-	"sort"
-	"strings"
-
 	"github.com/dave/dst"
 	"github.com/pkg/errors"
 	"k8s.io/klog/v2"
+	"sort"
```

Comment:
minor: The reordering here seems incorrect?

I thought our pattern was:
```
stdlib

external

local
```

Which is what this originally had at the top?

Reply:
I think you can configure goland to do it for you -- I don't think gofmt does this one automatically but I could be wrong

Reply:
I already have GoLand configured to automatically apply **go fmt** when saving. Don't know what's going on here.